app/scripts/modules/amazon/src/serverGroup/configure/wizard/AmazonCloneServerGroupModal.tsx

```diff
@@ -141,6 +141,17 @@ export class AmazonCloneServerGroupModal extends React.Component<
     });
   };
 
+  private normalizeCommand = ({ tags }: IAmazonServerGroupCommand) => {
+    if (!tags) {
+      return;
+    }
+    Object.keys(tags).forEach(key => {
+      if (!key.length && !tags[key].length) {
+        delete tags[key];
+      }
+    });
```

Comment:
@jrsquared do you know whether this field needs reference equality for angular shenanigans once it hits the submit flow? I wrote it this way under that anxiety but if possible would love to generate a new `tags` object.

Reply:
I do not _believe_ so, but I think I left it this way under the same anxiety.

Reply:
glad to hear we're both entirely driven by FUD, will leave it as-is 😉